blog-posts/case-fc-bayern.md

```diff
@@ -0,0 +1,66 @@
+---
+path: "/blog/case-fc-bayern/"
+date: "2022-02-21"
+title: "Relaunch FC Bayern Website"
+featuredImage: images/fcb/hero.png
+author: Fabian Dietenberger
+authorSummary: "Frontend Engineer, Satellytes"
+seoMetaText: Our Approach to relaunch the Frontend of the Fc Bayern Website.
+teaserText: Our Approach to relaunch the Frontend of the Fc Bayern Website.
+---
+<sectionheader>
+ Our task was the frontend relaunch of the FC Bayern website. We took a holistic approach and created a design system with a solid set of components and guidelines to unify the look and feel of forthcoming digital experiences and introduced a headless architecture as a modern foundation on top of the existing CMS to be prepared for everything to come.
+</sectionheader>
+
+#### [www.fcbayern.com](www.fcbayern.com)
+
+<br><br>
+
+<stat title="Team Size" content="6 people"></stat>
+<stat title="Start" content="December 2018"></stat>
+<stat title="End" content="Ongoing"></stat>
+
+
+![](images/fcb/overview.png)
```

Comment:
It's not full-width (aka escaping the layout), but I don't see a simple way to achieve that in the blog post format. We could find a solution but only if we want to aim for full width images in blog posts in general. Otherwise I would suggest to keep it as is and develop the "case template" afterwards, that should support that type of image out of the box.